meta-resin-common/recipes-support/resin-vars/resin-vars/resin-vars

```diff
@@ -34,9 +34,17 @@ while [[ $# > 0 ]]; do
     shift
 done
 
+# Retrive resin-boot mountpoint
+BOOT_MOUNTPOINT="$(grep $(readlink -f /dev/disk/by-label/resin-boot) /proc/mounts | cut -d ' ' -f 2)"
+
+if [ -z "$BOOT_MOUNTPOINT" ]; then
+    echo "[ERROR] : Could not determine mount point for resin-boot partition"
+    exit 1
```

Comment:
I think you should exit here only if CONFIG_PATH is not set because you would have to use it from the non-existent mount point.. Or better, error out bellow when trying to set CONFIG_PATH to the default value.